src/book/01-team/04-history/01-team-history.mdx

```diff
@@ -66,3 +66,72 @@ NUbots made it to the quarter-finals of the competition. Overall we played 13 ga
 | Eliminations for Quarter-Finals (Penalty Shootout) | 3      | 1      | 1   | 0    | 0    |
 | Quarter-Finals                                     | 0      | 1      | 0   | 0    | 1    |
 | Drop-Ins                                           | -5     | 8      | 4   | 1    | 3    |
+
+## 2010: Singapore
+
+**Major hardware changes**
+
+- Upgraded to NAOv3: Intel ATOM 1.6GHz, 1GB Ram
+
+**Major software changes**
```

Comment:
We can remove the heading if there's no information.

```suggestion
```

Reply:
I thought about this then decided it's probably good to have so people know we didn't forget about it, there's just nothing to put in? Although I assume there would have been _some_ change to the software 🤔

Reply:
In the case of nothing to put in, we could explicitly add something like _No major software changes_. 

And yeah, there was probably some change to the software.